src/main/java/com/futurecollars/zad4/Circle.java

```diff
@@ -0,0 +1,17 @@
+package com.futurecollars.zad4;
+
+public class Circle extends Shape {
+    private final double r;
```

Comment:
r to nazwa matematyczna, w kodzie staramy się nazywać rzeczy bardziej opisowo, tutaj radius